backend/api/pipeline.proto

```diff
@@ -320,6 +320,15 @@ message Pipeline {
   repeated ResourceReference resource_references = 9;
 }
 
+enum PipelineSpecFormat {
```

Comment:
I think that we need to carry this enum when there is a pipeline_spec field. How about the following? (We can decide the naming as it needs)

```
message PipelineDefintion {
   enum PipelineSpecFormat {
     ...
   }

   string content = 0;

   PipelineSpecFormat pipeline_spec_format = 1;
}

```

And `PipelineDefintion` will replace `workflow_manifest` field, or become the structure of `PipelineSpec` message.

Reply:
Can we rename it to `PipelineSpecSchemaVersion`